### PR TITLE
Upgrade deps

### LINKED ISSUE #91: Fix vulnerabilities
I saw there are some packages with vulnerabilities (run `npm audit`).

I think we need to upgrade `copy-webpack-plugin`.

Happy to do the PR if needed

### PR DESCRIPTION
**Pull request by poteirard**
I had to upgrade a lot of libraries in order to fix all the vulnerabilities. Sorry if the PR is too big

Fixes #91 #92

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "dependencies": {
-    "js-cookie": "^2.2.1"
+    "js-cookie": "^2.2.1",
+    "prop-types": "^15.7.2"
```
Comment (poteirard) [inline package.json:13]: I added this as suggested in the official docs https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package

File: webpack.config.js [hunk 2]
```diff
@@ -1,32 +1,30 @@
-var path = require('path');
-var CopyWebpackPlugin = require('copy-webpack-plugin');
+var path = require("path");
```
Comment (poteirard) [inline webpack.config.js:1]: All this formating comes from git hooks > prettier.

File: webpack.config.js [hunk 3]
```diff
@@ -1,32 +1,30 @@
...
+    ],
   },
   externals: {
-    'react': 'commonjs react' // this line is just to use the React dependency of our parent-testing-project instead of using our own React.
+    react: "commonjs react", // this line is just to use the React dependency of our parent-testing-project instead of using our own React.
   },
-  plugins: [
-    new CopyWebpackPlugin([{ from: 'src/index.d.ts', to: 'index.d.ts' }])
-  ]
+  plugins: [new CopyWebpackPlugin({ patterns: [{ from: "src/index.d.ts", to: "index.d.ts" }] })],
```
Comment (poteirard) [inline webpack.config.js:29]: This is the new syntax according to the [official docs](https://github.com/webpack-contrib/copy-webpack-plugin)

**Comment by Mastermindzh** [comment]
Pr looks good. I knew the files weren't up to date with prettier so no worries.
HUGE package-json update as expected :stuck_out_tongue: 

I did not the prop-types thing as a dependency. Never used it like that. Guess I need to revisit a lot of projects :smile: 

Will merge this together with the other one after some tests etc.

Thanks again for the work!

**Pull request merged**